build.zig

```diff
@@ -284,6 +359,45 @@ const generic_src_files = [_][]const u8{
     "src/joystick/hidapi/SDL_hidapijoystick.c",
 };
 
+const android_src_files = [_][]const u8{
+    "src/core/android/SDL_android.c",
+
+    "src/audio/android/SDL_androidaudio.c",
+    "src/audio/openslES/SDL_openslES.c",
+    "src/audio/aaudio/SDL_aaudio.c",
+
+    "src/haptic/android/SDL_syshaptic.c",
+    "src/joystick/android/SDL_sysjoystick.c",
+    "src/locale/android/SDL_syslocale.c",
+    "src/misc/android/SDL_sysurl.c",
+    "src/power/android/SDL_syspower.c",
+    "src/filesystem/android/SDL_sysfilesystem.c",
+    "src/sensor/android/SDL_androidsensor.c",
+
+    "src/timer/unix/SDL_systimer.c",
+    "src/loadso/dlopen/SDL_sysloadso.c",
+
+    "src/thread/pthread/SDL_syscond.c",
+    "src/thread/pthread/SDL_sysmutex.c",
+    "src/thread/pthread/SDL_syssem.c",
+    "src/thread/pthread/SDL_systhread.c",
+    "src/thread/pthread/SDL_systls.c",
+    "src/render/opengles/SDL_render_gles.c", // use of undeclared identifier: glCheckFramebufferStatusOES
```

Comment:
What is this comment trying to communicate?

Reply:
IIRC, without this file, compilation or linking will fail with `use of undeclared identifier: glCheckFramebufferStatusOES`